solver/src/http_solver/model.rs

```diff
@@ -43,6 +43,7 @@ pub struct UniswapModel {
 pub struct TokenInfoModel {
     pub decimals: Option<u32>,
     pub external_price: Option<f64>,
+    pub normalize_priority: Option<u64>,
```

Comment:
Why Option if it's always set?

Reply:
The http solver API doesn't require it (it defaults to zero if it's not set).